DeathNotes.cs

```diff
@@ -381,9 +381,19 @@ private string GetEntityName(BaseEntity entity, CombatEntityType combatEntityTyp
                 case CombatEntityType.Helicopter:
                     return "Helicopter";
 
+                case CombatEntityType.Other:
+                    return "Other";
+
+                case CombatEntityType.None:
+                    return "None";
+

```

Comment:
These are redundant, as the last line in this method should already handle this.
Notably though, the same is the case for the existing Helicopter case above. You can also remove that please.